config/fedora.yml

```diff
@@ -0,0 +1,20 @@
+development:
+  user: fedoraAdmin
+  password: fedoraAdmin
+  url: http://localhost:8984/rest
+  base_path: /dev
+test:
+  user: fedoraAdmin
+  password: fedoraAdmin
+  url: http://localhost:8986/rest
+  base_path: /test
+production:
+  user: <%= ENV["PLUM_FEDORA_USER"] || "fedoraAdmin" %>
```

Comment:
Oh. I guess. It never actually uses this file, staging just won't spin up without it